### PR TITLE
Updated README to reflect changes in core team approach

### PR DESCRIPTION
**Pull request by ColinEberhardt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,26 +22,26 @@ The AI readiness group is at the very early formation stages, as a result, the g
...
- - Madhu Coimbatore, Morgan Stanley, Executive Director, AI Platform Development
- - Andrew Martin, CEO, ControlPlane
- - Oleg Blokhin, Executive General Manager, Provectus
- - Vicente Herrera Garcia, Principal Consultant, ControlPlane
- - James Heward, Principal Architect, Scott Logic
- - Mehak Mehta, AIMLOps Lead, Morgan Stanley
-
-We welcome further core team members.
+
+The group meets each week, with the focus of the meeting alternating each fortnight:
```
Comment (caradelia) [inline README.md:42]: It looks that the core team meets every 2 weeks per Luca's invite not weekly.

File: README.md [hunk 2]
```diff
@@ -22,26 +22,26 @@ The AI readiness group is at the very early formation stages, as a result, the g
...
 
-## Membership
+## Membership 
 
 The AI Readiness is FINOS members only while we are in the formation stage. To join the group, please contact membersuccess@finos.org 
 
 We use the following channels:
  - Group email distribution list: ai-readiness-private@lists.finos.org
- - Meetings follow Chatham House rules, with no formal minutes, however, actions and conclusions will be shared via the email distribution list.
+ - Meetings follow Chatham House rules, where notes are taken without attribution. Meeting agenda and notes are [recorded on GitHub](https://github.com/finos/ai-readiness/issues?q=is%3Aissue+is%3Aopen+label%3Ameeting).
```
Comment (caradelia) [inline README.md:31]: [Chatham House Rule] (https://en.wikipedia.org/wiki/Chatham_House_Rule)

File: README.md [hunk 3]
```diff
@@ -22,26 +22,26 @@ The AI readiness group is at the very early formation stages, as a result, the g
  - **Leverage existing frameworks** -  financial services organisations already have robust frameworks in place for dealing with technology risk. We’ll look to leverage these framework, or approaches, where appropriate.
```
Comment (caradelia) [inline README.md:22]: frameworks

File: README.md [hunk 4]
```diff
@@ -22,26 +22,26 @@ The AI readiness group is at the very early formation stages, as a result, the g
...
  - **Threat modelling** - for our selected use case(s), we’ll use threat modelling techniques to map out the challenge, then identify suitable mitigation
```
Comment (caradelia) [inline README.md:23]: This is pretty broad, which is fine, but might want to include that the core team is looking to further define scope.(For example, are we looking to secure the framework including data sources or just the system of record.)

File: README.md (hunk 1 above)
Comment (lucaborella89) [reply README.md:42]: @ColinEberhardt FINOS will add the "Governance framework working session" on the FINOS Community Calendar, biweekly on Wednesday @3pm BST, which will be alternating with a bi-weekly "General SIG call" on Tuesday @4pm BST as per your update.

**Pull request merged**